wasm/store.go

```diff
@@ -868,15 +870,25 @@ type HostFunctionCallContext struct {
 	ctx context.Context
 	// Memory is the currently used memory instance at the time when the host function call is made.
 	Memory *MemoryInstance
+	// Used in `RandomGet` method
+	Randomizer *rand.Rand
```

Comment:
we should _not_ define this here because this results in introducing WASI thingy into Wasm space. 
Let's have this in wasm.api struct and have an option to control this